client/goal.c

```diff
@@ -619,3 +622,55 @@ TDNFGoalGetAllResultsIgnoreNoData(
     }
     goto cleanup;
 }
+
+uint32_t
+TDNFSolvAddPkgLocks(
+    PTDNF pTdnf,
+    Queue* pQueueJobs,
+    Pool *pPool
+    )
+{
+    uint32_t dwError = 0;
+    char **ppszPackages = NULL;
+    char *pszPkg = NULL;
+    Id idPkg;
```

Comment:
we can reduce the scope of few of these variables by declaring them inside loop where required.

Reply:
Done.